test/integration/controlplane/transformation/secrets_transformation_test.go

```diff
@@ -66,7 +84,10 @@ resources:
   - resources:
     - secrets
     providers:
-    - identity: {}
+    - aesgcm:
+        keys:
+        - name: key1
+          secret: dXBlcnByZGVscHJkZWxwCg==
```

Comment:
This is the `identityConfigYAML`, why are we changing the provider to `aesgcm` instead of using `aesGCMConfigYAML`?

Reply:
(this seems unexpected to me)

Reply:
yes, it's fixed now. the test configures encryption with `aesGCMConfigYAML` and then it replaces the config entirely with `aesCBCConfigYAML` resulting in a `no matching prefix found` error